docs/source/tracking/index.rst

```diff
@@ -0,0 +1,311 @@
+
+By configuring these components properly, you can create an MLflow Tracking environment suitable for your team's development workflow.
+The following diagram and table show a few common setups for the MLflow Tracking environment.
+
+.. figure:: ../_static/images/tracking/tracking-setup-overview.png
+    :align: center
+    :figwidth: 900
+
+    Different ways to set up the MLflow environment for tracking your experiments.
+
+.. list-table::
+    :widths: 45 15 20 20
+    :header-rows: 1
+
+    * - Scenario
+      - Use Case
+      - Description
+      - Tutorial
+
+    * - **1. Localhost (default)**
+      - Solo development
+      - By default, MLflow records metadata and artifacts for each run to a local directory, ``mlruns``. This is the simplest way to get started with MLflow Tracking, without setting up any external server, database, and storage.
+      - `QuickStart <../getting-started/intro-quickstart/index.html>`_
+
+    * - **2. Local Tracking with Local Database**
+      - Solo development
+      - The MLflow client can interface with a SQLAlchemy-compatible database (e.g., SQLite, PostgreSQL, MySQL) for the `backend <backend-stores.html>`_. Saving metadata to a database allows you cleaner management of your experiment data while skipping the effort of setting up a server.
+      - `Tracking Experiments with Local Database <tutorials/local-database.html>`_
+
+    * - **3. Remote Tracking with** :ref:`MLflow Tracking Server <tracking_server>`
+      - Team development
+      - MLflow Tracking Server can be configured with an artifacts HTTP proxy, passing artifact requests through the tracking server to store and retrieve artifacts without having to interact with underlying object store services. This is particularly useful for team development scenarios where you want to store artifacts and experiment metadata in a shared location with proper access control.
+      - `Remote Experiment Tracking with MLflow Tracking Server <tutorials/remote-server.html>`_
+
+.. _other-tracking-setup:
+
+Other Configuration with :ref:`MLflow Tracking Server <tracking_server>`
+------------------------------------------------------------------------
+MLflow Tracking Server provides customizability for other special use cases. Please follow `Remote Experiment Tracking with MLflow Tracking Server <tutorials/remote-server.html>`_ for
+learning the basic setup and continue to the following materials for advanced configurations to meet your needs.
+
+.. |image-local-server| image:: ../_static/images/tracking/tracking-setup-local-server.png
+    :align: middle
+    :width: 50px
+    :alt: Run MLflow Tracking Server on localhost
+
+.. |image-no-proxy| image:: ../_static/images/tracking/tracking-setup-no-serve-artifacts.png
+    :align: middle
+    :width: 50px
+    :alt: Bypass Tracking Server proxy for artifacts access
+
+.. |image-artifact-only| image:: ../_static/images/tracking/tracking-setup-artifacts-only.png
+    :align: middle
+    :width: 50px
```

Comment:
Can we set % of DOM display element instead of fixed size just to make sure that screen resizing scales correctly?

Reply:
Absolutely, should've replaced the test size.